assets/scripts/admin/questionEditor.js

```diff
@@ -929,7 +929,7 @@ $(document).on('ready pjax:scriptcomplete', function () {
     // TODO: Doc answers
     const answers = {};
     const lsrows = $('#quickaddarea').val().split('\n');
-    const allrows = $closestTable.find('tr').length;
+    const allrows = $closestTable.find('tbody tr').length;
```

Comment:
? needed ? A row is a tr no ?

Reply:
The header rows are also TR. We only need the body rows. That's why we filter the body in the CSS selector

Reply:
Oups : it's a selector : totaly right , sorry !